ecosystem/typescript/sdk_v2/src/transactions/instances/transactionPayload.ts

```diff
@@ -158,12 +191,15 @@ export class EntryFunction {
     const type_args = deserializer.deserializeVector(TypeTag);
 
     const length = deserializer.deserializeUleb128AsU32();
-    const list: Array<Uint8Array> = [];
+    const list: Array<Serializable> = new Array<MoveVector<U8>>();
+
     for (let i = 0; i < length; i += 1) {
-      list.push(deserializer.deserializeBytes());
+      const vec = MoveVector.deserialize(deserializer, U8);
+      list.push(new MoveVector(vec.values.splice(1)));
```

Comment:
The new change is in my other PR, but keep in mind this is actually wrong. That's my bad, heh, but basically there's no way to correctly represent this in terms of being able to `serialize => deserialize => serialize` unless we use some form of `FixedBytes`.

Reply:
updated